dapp/src/components/buySell/SwapHomepage.js

```diff
@@ -587,11 +596,16 @@ const SwapHomepage = ({
           </a>
           <button
             //disabled={formHasErrors || buyFormHasWarnings || !totalOUSD}
-            className={`btn-blue buy-button mt-2 mt-md-0 w-100`}
+            className={`btn-dark buy-button mt-2 mt-md-0 w-100`}
             disabled={!selectedSwap || formHasErrors}
             onClick={onBuyNow}
           >
-            {fbt('Swap', 'Swap')}
+            {!formHasErrors
+              ? fbt('Swap', 'Swap')
+              : fbt(
+                  'Insufficient balance',
```

Comment:
Would be better to not hardcode the error and display the actual error here. Not sure the correct value could already be present in `formError` variable?